Example/android/app/build.gradle

```diff
@@ -139,11 +139,11 @@ dependencies {
 
 apply plugin: 'com.github.node-gradle.node'
 
-task syncArchs(type: NodeTask) {
-    group = 'Build'
-    description = 'Run sync beetwen Paper and Fabric arch'
-    yarn_install
-    script = file('../../../scripts/codegen-sync-archs.js')
-}
-
-tasks['preBuild'].dependsOn(syncArchs)
+//task syncArchs(type: NodeTask) {
+//    group = 'Build'
+//    description = 'Run sync beetwen Paper and Fabric arch'
+//    yarn_install
+//    script = file('../../../scripts/codegen-sync-archs.js')
+//}
+//
+//tasks['preBuild'].dependsOn(syncArchs)
```

Comment:
Tested it. No idea why this causes problems & don't want to spend time right now on it. Let's leave it be (commented out) for now. I think this will sort out when we upgrade to 0.78.